en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+# Develop an API Proxy from Git Repository
+
+An API proxy acts as an intermediary between an existing API and Choreo, intercepting all requests made to the API. It also functions as a managed API, allowing you to apply essential API management features such as security policies and rate limiting.
+
+In this guide, you will:
+
+- Create an API proxy component to expose an existing API.
+- Deploy the API proxy.
+- Test the API proxy to verify its functionality.
+- Manage the API.
+- Consume the API.
+
+## Prerequisites
+
+- If you're signing in to the Choreo Console for the first time, create an organization:
+    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in using your preferred method.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use.
+    4. Click **Create**.
+
+    This creates the organization and opens the **Project Home** page of the default project created for you.
+
+## Step 1: Create an API proxy
+
+To create an API proxy, you can either choose from a github repository, upload an OpenAPI specification or provide an OpenAPI specification URL. In this guide, you will choose the API definition from a Git source like Github repository.
```

Comment:
Make sure to use the correct capitalization for "GitHub" in all occurrences.

Suggestion:
You can create an API proxy either by selecting the source from a GitHub repository, uploading an OpenAPI specification file, or providing an OpenAPI specification URL. This guide demonstrates how to create an API proxy using a GitHub repository as the source.

Reply:
`Also, at the moment users can only select from GitHub right? Other Git repositories are not supported as of now isn't it?`
Yes.